### ISSUE TITLE
Add common sense basic items for easy logging

### ISSUE THREAD

**Issue by PropFault** [davidhealey/waistline#872]
The biggest issue I have with this app is that is it very difficult for me, someone who doesn't live in america, to log basic foods. Most vegetables here either do not have barcodes or are not in OFF. It would be very nice to have basic groceries, such as common vegetables, eggs etc. just be available in the app by default. 

As is right now its much easier to log pre-made foods than own recipes for the simple fact that basic ingredients do not have good representation, especially when you do not have the ability to scan a barcode. Typing in "carrot" into the search bar does not yield regular carrots. You will find frozen carrots, processed salads with carrots etc. but not just 100g of fresh carrot. 

I feel this is a major painpoint for me at least and it would be nice to just have the app ship with a predefined list of the most common "basic" groceries that can reasonably not be expected to be in OFF or even have a barcode.

**Comment by EmilJunker** [comment]
You can expand the in-app search functionality to include generic and non-branded foods via `Settings > Integration > USDA API`. Admittedly, the integration isn't perfect and only English search queries are supported, but it's much better than having to add foods manually.

You can find more information in the user guide: 
https://github.com/davidhealey/waistline/blob/master/FAQ.md#how-can-i-search-for-foods-online

**Comment by darkenedcolossal** [comment]
> You can expand the in-app search functionality to include generic and non-branded foods via `Settings > Integration > USDA API`. Admittedly, the integration isn't perfect and only English search queries are supported, but it's much better than having to add foods manually.
> 
> You can find more information in the user guide: https://github.com/davidhealey/waistline/blob/master/FAQ.md#how-can-i-search-for-foods-online

This helped, but only after adding a free API key. Which was easy to receive. But it is confusing that you can activate USDA Search, but it just won't work without API key. No warning...
